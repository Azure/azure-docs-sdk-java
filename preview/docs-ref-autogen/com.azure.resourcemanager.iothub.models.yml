### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.iothub.models"
fullName: "com.azure.resourcemanager.iothub.models"
name: "com.azure.resourcemanager.iothub.models"
summary: "Package containing the data models for IotHubClient. Use this API to manage the IoT hubs in your Azure subscription."
classes:
- "com.azure.resourcemanager.iothub.models.AuthenticationType"
- "com.azure.resourcemanager.iothub.models.Capabilities"
- "com.azure.resourcemanager.iothub.models.CertificateBodyDescription"
- "com.azure.resourcemanager.iothub.models.CertificateProperties"
- "com.azure.resourcemanager.iothub.models.CertificatePropertiesWithNonce"
- "com.azure.resourcemanager.iothub.models.CertificateVerificationDescription"
- "com.azure.resourcemanager.iothub.models.CloudToDeviceProperties"
- "com.azure.resourcemanager.iothub.models.EndpointHealthDataListResult"
- "com.azure.resourcemanager.iothub.models.EndpointHealthStatus"
- "com.azure.resourcemanager.iothub.models.EnrichmentProperties"
- "com.azure.resourcemanager.iothub.models.ErrorDetails"
- "com.azure.resourcemanager.iothub.models.ErrorDetailsException"
- "com.azure.resourcemanager.iothub.models.EventHubConsumerGroupsListResult"
- "com.azure.resourcemanager.iothub.models.EventHubProperties"
- "com.azure.resourcemanager.iothub.models.ExportDevicesRequest"
- "com.azure.resourcemanager.iothub.models.FailoverInput"
- "com.azure.resourcemanager.iothub.models.FallbackRouteProperties"
- "com.azure.resourcemanager.iothub.models.FeedbackProperties"
- "com.azure.resourcemanager.iothub.models.GroupIdInformationProperties"
- "com.azure.resourcemanager.iothub.models.ImportDevicesRequest"
- "com.azure.resourcemanager.iothub.models.IotHubCapacity"
- "com.azure.resourcemanager.iothub.models.IotHubDescriptionListResult"
- "com.azure.resourcemanager.iothub.models.IotHubLocationDescription"
- "com.azure.resourcemanager.iothub.models.IotHubProperties"
- "com.azure.resourcemanager.iothub.models.IotHubQuotaMetricInfoListResult"
- "com.azure.resourcemanager.iothub.models.IotHubReplicaRoleType"
- "com.azure.resourcemanager.iothub.models.IotHubSku"
- "com.azure.resourcemanager.iothub.models.IotHubSkuDescriptionListResult"
- "com.azure.resourcemanager.iothub.models.IotHubSkuInfo"
- "com.azure.resourcemanager.iothub.models.IpFilterRule"
- "com.azure.resourcemanager.iothub.models.JobResponseListResult"
- "com.azure.resourcemanager.iothub.models.JobType"
- "com.azure.resourcemanager.iothub.models.MatchedRoute"
- "com.azure.resourcemanager.iothub.models.MessagingEndpointProperties"
- "com.azure.resourcemanager.iothub.models.Name"
- "com.azure.resourcemanager.iothub.models.OperationDisplay"
- "com.azure.resourcemanager.iothub.models.OperationInputs"
- "com.azure.resourcemanager.iothub.models.OperationListResult"
- "com.azure.resourcemanager.iothub.models.PrivateEndpoint"
- "com.azure.resourcemanager.iothub.models.PrivateEndpointConnectionProperties"
- "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionStatus"
- "com.azure.resourcemanager.iothub.models.PublicNetworkAccess"
- "com.azure.resourcemanager.iothub.models.RouteCompilationError"
- "com.azure.resourcemanager.iothub.models.RouteErrorPosition"
- "com.azure.resourcemanager.iothub.models.RouteErrorRange"
- "com.azure.resourcemanager.iothub.models.RouteErrorSeverity"
- "com.azure.resourcemanager.iothub.models.RouteProperties"
- "com.azure.resourcemanager.iothub.models.RoutingEndpoints"
- "com.azure.resourcemanager.iothub.models.RoutingEventHubProperties"
- "com.azure.resourcemanager.iothub.models.RoutingMessage"
- "com.azure.resourcemanager.iothub.models.RoutingProperties"
- "com.azure.resourcemanager.iothub.models.RoutingServiceBusQueueEndpointProperties"
- "com.azure.resourcemanager.iothub.models.RoutingServiceBusTopicEndpointProperties"
- "com.azure.resourcemanager.iothub.models.RoutingSource"
- "com.azure.resourcemanager.iothub.models.RoutingStorageContainerProperties"
- "com.azure.resourcemanager.iothub.models.RoutingStorageContainerPropertiesEncoding"
- "com.azure.resourcemanager.iothub.models.RoutingTwin"
- "com.azure.resourcemanager.iothub.models.RoutingTwinProperties"
- "com.azure.resourcemanager.iothub.models.SharedAccessSignatureAuthorizationRuleListResult"
- "com.azure.resourcemanager.iothub.models.StorageEndpointProperties"
- "com.azure.resourcemanager.iothub.models.TagsResource"
- "com.azure.resourcemanager.iothub.models.TestAllRoutesInput"
- "com.azure.resourcemanager.iothub.models.TestResultStatus"
- "com.azure.resourcemanager.iothub.models.TestRouteInput"
- "com.azure.resourcemanager.iothub.models.TestRouteResultDetails"
- "com.azure.resourcemanager.iothub.models.UserSubscriptionQuota"
enums:
- "com.azure.resourcemanager.iothub.models.AccessRights"
- "com.azure.resourcemanager.iothub.models.IotHubNameUnavailabilityReason"
- "com.azure.resourcemanager.iothub.models.IotHubScaleType"
- "com.azure.resourcemanager.iothub.models.IotHubSkuTier"
- "com.azure.resourcemanager.iothub.models.IpFilterActionType"
- "com.azure.resourcemanager.iothub.models.JobStatus"
interfaces:
- "com.azure.resourcemanager.iothub.models.CertificateDescription"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.Definition"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages.Blank"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages.WithCertificate"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages.WithIfMatch"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.Update"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.UpdateStages"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.UpdateStages.WithCertificate"
- "com.azure.resourcemanager.iothub.models.CertificateDescription.UpdateStages.WithIfMatch"
- "com.azure.resourcemanager.iothub.models.CertificateListDescription"
- "com.azure.resourcemanager.iothub.models.CertificateWithNonceDescription"
- "com.azure.resourcemanager.iothub.models.Certificates"
- "com.azure.resourcemanager.iothub.models.EndpointHealthData"
- "com.azure.resourcemanager.iothub.models.EventHubConsumerGroupInfo"
- "com.azure.resourcemanager.iothub.models.GroupIdInformation"
- "com.azure.resourcemanager.iothub.models.IotHubDescription"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.Definition"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.Blank"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithIfMatch"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithSku"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.DefinitionStages.WithTags"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.Update"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.UpdateStages"
- "com.azure.resourcemanager.iothub.models.IotHubDescription.UpdateStages.WithTags"
- "com.azure.resourcemanager.iothub.models.IotHubNameAvailabilityInfo"
- "com.azure.resourcemanager.iothub.models.IotHubQuotaMetricInfo"
- "com.azure.resourcemanager.iothub.models.IotHubResources"
- "com.azure.resourcemanager.iothub.models.IotHubSkuDescription"
- "com.azure.resourcemanager.iothub.models.IotHubs"
- "com.azure.resourcemanager.iothub.models.JobResponse"
- "com.azure.resourcemanager.iothub.models.Operation"
- "com.azure.resourcemanager.iothub.models.Operations"
- "com.azure.resourcemanager.iothub.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.iothub.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.iothub.models.PrivateLinkResources"
- "com.azure.resourcemanager.iothub.models.PrivateLinkResourcesOperations"
- "com.azure.resourcemanager.iothub.models.RegistryStatistics"
- "com.azure.resourcemanager.iothub.models.ResourceProviderCommons"
- "com.azure.resourcemanager.iothub.models.SharedAccessSignatureAuthorizationRule"
- "com.azure.resourcemanager.iothub.models.TestAllRoutesResult"
- "com.azure.resourcemanager.iothub.models.TestRouteResult"
- "com.azure.resourcemanager.iothub.models.UserSubscriptionQuotaListResult"
metadata: {}
package: "com.azure.resourcemanager.iothub.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-iothub:1.0.0-beta.1
