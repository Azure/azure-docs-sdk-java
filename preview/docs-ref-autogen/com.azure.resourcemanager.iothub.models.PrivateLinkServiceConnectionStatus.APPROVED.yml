### YamlMime:JavaMember
uid: "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.iothub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.iothub.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-iothub:1.0.0-beta.1
